internetarchive/utils.py

```diff
@@ -363,7 +365,7 @@ def is_valid_metadata_key(name):
     return bool(re.fullmatch(r'[A-Za-z][.\-0-9A-Za-z_]+(?:\[[0-9]+\])?', name))
 
 
-def merge_dictionaries(dict0, dict1, keys_to_drop=None):
+def merge_dictionaries(dict0: dict, dict1: dict, keys_to_drop=None) -> dict:
```

Comment:
`dict0: Mapping, dict1: Mapping, keys_to_drop: Optional[Iterable] = None`

Reply:
Mypy: internetarchive/utils.py:397: error: "Mapping[Any, Any]" has no attribute "copy"

Reply:
Some of those interfaces are defined ratherstrangely ...